src/main/scala/org/broadinstitute/dsde/agora/server/webservice/routes/RouteHelpers.scala

```diff
@@ -38,14 +39,12 @@ trait NamespacePermissionsRouteHelper extends BaseRoute {
   def completeNamespacePermissionsGet(context: RequestContext, entity: AgoraEntity, username: String, aclHandler: Props) = {
     addUserIfNotInDatabase(username)
     val message = ListNamespacePermissions(context, entity, username)
-    perRequest(context, aclHandler, message)
```

Comment:
add these back to the code. and rename the aclHandler variable to PermissionHandler -- forgot to clean that up in my branch.